### PR TITLE
[Wallet] Differentiate dollars and gold transactions

### LINKED ISSUE #899: Users SBAT differentiate Celo gold and dollar transactions
### Expected Behavior

Users SBAT to tell which transactions are for gold and display the corresponding local currency amounts (if local currency is to be shown for gold transactions).

Gold Transactions (except for exchanges) should be in the 'Gold' Tab Feed only
Exchanges should be in both feeds
Dollar transactions should be in only the 'Wallet' Feed
'Received' transactions(balance increase) should have the amount colored either green or gold, depending on the currency

### Current Behavior

Faucet transactions that give the user Celo Gold display the same as Celo Dollars including the local currency amount.  

Top is Celo Dollars, bottom is Celo Gold.

Screenshot_2019-09-10-09-48-53

Color from getCurrencyStyles is no longer used:

https://github.com/celo-org/celo-monorepo/blob/dfdc3e8b26e98aa294b27e2b5621c184488a10db/packages/mobile/src/transactions/TransferFeedItem.tsx#L76-L78

https://github.com/celo-org/celo-monorepo/blob/dfdc3e8b26e98aa294b27e2b5621c184488a10db/packages/mobile/src/transactions/TransferFeedItem.tsx#L211-L221

### PR DESCRIPTION
**Pull request by jeanregisser**
### Description

Gold transactions could not be differentiated from dollars transactions in the wallet activity. This PR moves them to the Gold tab and also makes the exchange transactions display gold amounts when displayed in the Gold tab.

### Tested

Updated tests.

**Before**

Screenshot 2019-09-17 at 10 15 58

**After**

Screenshot 2019-09-17 at 10 13 57

### Other changes

- Renamed "Activity" to "Gold Activity" in the gold tab

### Related issues

- Fixes #899

### Backwards compatibility

Yes.

### CODE REVIEW

File: packages/mobile/src/transactions/ExchangeFeedItem.tsx [hunk 1]
```diff
@@ -36,6 +46,10 @@ export function ExchangeFeedItem(props: Props) {
   const outCurrency = resolveCurrency(outSymbol)
   const dollarAmount = inCurrency === CURRENCY_ENUM.DOLLAR ? inValue : outValue
```
Comment (jmrossy) [inline packages/mobile/src/transactions/ExchangeFeedItem.tsx]: A lot of repeated ternary conditions here. Consider reorganizing this similar to what I did here:
https://github.com/celo-org/celo-monorepo/blob/master/packages/mobile/src/exchange/ExchangeTradeScreen.tsx#L188

File: packages/mobile/src/transactions/ExchangeFeedItem.tsx [hunk 2]
```diff
@@ -59,32 +73,33 @@ export function ExchangeFeedItem(props: Props) {
...
+                  : {
+                      ...fontStyles.activityCurrencyReceived,
+                      color: showGoldAmount ? colors.celoGold : colors.celoGreen,
+                    },
                 styles.amount,
               ]}
             >
-              {dollarDirection}
-              {getMoneyDisplayValue(dollarAmount)}
+              {amountDirection}
```
Comment (jmrossy) [inline packages/mobile/src/transactions/ExchangeFeedItem.tsx:114]: Should we include this as a param for `getMoneyDisplayValue` and let it add the amount direction as necessary?

**Review by jmrossy — APPROVED** [review]
Approved with two optional comments

File: packages/mobile/src/transactions/ExchangeFeedItem.tsx (hunk 2 above)
Comment (jeanregisser) [reply packages/mobile/src/transactions/ExchangeFeedItem.tsx:114]: I'd say let's leave this for now and reconsider if we see this pattern repeat.

**Pull request merged**